.github/actions/setup-rubygems.org/action.yml

```diff
@@ -23,7 +23,7 @@ runs:
       if: github.secret_source != 'None' && inputs.install-avo-pro == 'true'
       run: |
         printf "BUNDLE_WITH=avo\nRAILS_GROUPS=avo\n" >> $GITHUB_ENV
-    - uses: ruby/setup-ruby@52753b7da854d5c07df37391a986c76ab4615999 # v1.191.0
+    - uses: ruby/setup-ruby@c6d4d2ab97f832d0fea49200f14c2441e56fd0a6 # v1.200.0
```

Comment:
The [released](https://github.com/ruby/setup-ruby/releases/tag/v1.200.0) commit for 1.200.0 is `217c988b8c2bf2bacb2d5c78a7e7b18f8c34daed`

Could we use that instead?